commands/adminsonline.js

```diff
@@ -22,18 +43,32 @@ module.exports = {
 
             rcon.on("connect", () => console.log("connected"));
             rcon.on("authenticated", () => console.log("authenticated"));
+            rcon.on("error", () => console.log("errors"));
             rcon.on("end", () => console.log("end"));
 
-            await rcon.connect();
-
-            let responses1 = await rcon.send(rconToSend);
+            try {	// tests to see if the connection works, if not fails to catch block 
 
-            msg.channel.send(responses1);
+                await rcon.connect();
+                const responses = await rcon.send(rconToSend);
+                rcon.end();
+                if (responses === void 0) { return console.log('fail') }; // checks to see if the reply is completly void - if so then returns without running, posts in console
```

Comment:
might wanna  change this as void 0 is not that nice but its not blocking

Reply:
blocking as you do not need to change this

Reply:
```suggestion
                if (responses) { return console.log('fail') }; // checks to see if the reply is completly void - if so then returns without running, posts in console
```
migth work too

Reply:
```suggestion
                if (!responses) { return console.log('fail') }; // checks to see if the reply is completly void - if so then returns without running, posts in console
```
Should it not be !responses 
?

Reply:
oh yeah